whois-api/src/main/java/net/ripe/db/whois/api/rdap/WhoisRdapService.java

```diff
@@ -215,14 +215,18 @@ private Response lookupWithRedirectUrl(final HttpServletRequest request, final S
     }
 
     private Response lookupForAutNum(final HttpServletRequest request, final String key) {
-        if (isRedirect(AUT_NUM, key) && !rdapRequestValidator.isReservedAsNumber(key)) {
-            return redirect(getRequestPath(request), AUT_NUM, key);
-        }
+        try {
+            if (isRedirect(AUT_NUM, key) && !rdapRequestValidator.isReservedAsNumber(key)) {
+                return redirect(getRequestPath(request), AUT_NUM, key);
+            }
```

Comment:
Incase of a redirect do we need to throw autnum exception? It will add an flat model which does not make sense as it is redirect to some other url and we are not sure which hierarchy they implemented it

Reply:
either a 200 or a 404 we need to set that conformance:

We should receive "nro_rdap_profile_asn_flat_0" because it is undelegated and we are using flat model (just for 404 on ASN) 

To signify use of the flat model, the string literal 
   "nro_rdap_profile_asn_flat_0" MUST be included as an rdapConformance 
   value in each response.

Reply:
lookup_autnum_not_found_still_flat_conformance -> 404 with the conformance (for Autum)
lookup_org_error_correct_conformance -> 404 without conformance (without Autnum)